### PR TITLE
Add input "URL"

### PR DESCRIPTION
**Pull request by jsDotx3**
Add input url for "og_image" setting

Signed-off-by: José <jsdotx3@gmail.com>

### CODE REVIEW

File: Themes/default/Themes.template.php [hunk 1]
```diff
@@ -418,6 +418,11 @@ function template_set_options()
 				echo '
 						<input type="number"', $min . $max . $step;
 			}
+			else if (isset($setting['type']) && $setting['type'] == 'url')
+			{
+			echo'
+			<input type="url"';
```
Comment (live627) [inline Themes/default/Themes.template.php]: Please keep the indenting uniform with the rest off the file.

**Review by live627 — CHANGES_REQUESTED** [review]

File: Themes/default/Themes.template.php (hunk 1 above)
Comment (jsDotx3) [reply Themes/default/Themes.template.php]: Ready, is good?

**Comment by XinYenFon** [comment]
You have to sign-off each commit not just first one.

**Comment by live627** [comment]
Squashed to single commit with the sign.

**Pull request merged**